src/components/Form/Input/Input.tsx

```diff
@@ -20,7 +22,13 @@ export default function Input({
     case 'text':
       return <BaseInput {...otherProps} />
     case 'currency':
-      return <CurrencyInput {...otherProps} />
+      return (
+        <CurrencyInput
```

Comment:
We don't need pass min and max here, because this props are present on otherProps object, so you can access inside the CurrencyInput component